pkg/cri/sbserver/helpers.go

```diff
@@ -46,6 +46,8 @@ import (
 	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
 )
 
+// TODO: Move common helpers for sbserver and podsandbox to a dedicated package once basic services are functinal.
```

Comment:
nit some of the TODO's it is obvious what the project is.. some could use (sbserver/sandboxed) or some other long running reference...

Reply:
I'll update those in follow up.